src/tools/pmempool/rm.c

```diff
@@ -49,8 +49,7 @@
 #include "set.h"
```

Comment:
Why can't we make pmempool rm command use libpmempool_rm function? Right now this is effectively duplicated code.